### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by me97esn**

### CODE REVIEW

File: packages/t2l-frontend/src/screens/wizard/PreviewStep.tsx [hunk 1]
```diff
@@ -154,18 +156,37 @@ export default function PreviewStep({
...
+              Transferrable
+              <p className="tooltip">
+                ⓘ
+                <span className="tooltiptext">
+                  Transferrable means that everything seems to be in order to
+                  transfer the grade to Ladok: The student exists in the
+                  corresponding place in Ladok, has no previous grade above F,
+                  and the grade is not already transferred.
+                </span>
+              </p>
```
Comment (exacs) [inline packages/t2l-frontend/src/screens/wizard/PreviewStep.tsx:169]: This is not accessible according to WCAG 2.1. due to the following reasons:

- [SC 2.5.3. Label in Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html) The tooltip (ⓘ) does not have an accessible name, i.e., a human-readable label like "more details" or "help"
- [SC 2.1.1. Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard) The icon is not focusable via keyboard nor actionable
- [SC 1.4.13. Content on Hover or Focus](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html) If tooltip works "on hover" (and not "on click") it must also comply with more requirements

You can read about the guidelines in Swedish here:

- https://www.digg.se/webbriktlinjer/alla-webbriktlinjer/se-till-att-text-pa-knappar-och-kontroller-overensstammer-med-maskinlasbara-namn
- https://www.digg.se/webbriktlinjer/alla-webbriktlinjer/all-funktionalitet-ska-kunna-anvandas-med-tangentbord
- https://www.digg.se/webbriktlinjer/alla-webbriktlinjer/innehall-som-visas-vid-hover-eller-fokus

File: packages/t2l-frontend/src/screens/wizard/PreviewStep.tsx [hunk 2]
```diff
@@ -154,18 +156,37 @@ export default function PreviewStep({
...
-            </tr>
-          ))}
+          {gradesWithStatus
+            .filter(
+              (grade) =>
+                !nonParticipants.find(
+                  (nonParticipant) =>
+                    grade.student.id === nonParticipant.student.id
+                )
+            )
```
Comment (exacs) [inline packages/t2l-frontend/src/screens/wizard/PreviewStep.tsx:181]: There is a simpler logic for "non participants" in [Lines 92-95 of this file](https://github.com/KTH/transfer-to-ladok/pull/32/files#diff-bce28dfe2443fc7f662388a180143b9a901258a1fa796ac24044aafd3d934b1dR92-R95). Why not use it?

**Review by exacs — CHANGES_REQUESTED** [review]

File: packages/t2l-frontend/src/screens/wizard/PreviewStep.tsx (hunk 2 above)
Comment (me97esn) [reply packages/t2l-frontend/src/screens/wizard/PreviewStep.tsx:181]: Good point, I'll change it

File: packages/t2l-frontend/src/screens/wizard/PreviewStep.tsx (hunk 1 above)
Comment (me97esn) [reply packages/t2l-frontend/src/screens/wizard/PreviewStep.tsx:169]: Ahh, good catch! Of course we should make it accessible.
I tried to see if there was a tooltip/Modal in KTH Style, but couldn't find any. Do you know if there is any component available for this use case?

Comment (exacs) [reply packages/t2l-frontend/src/screens/wizard/PreviewStep.tsx:169]: No. Unfortunately, we don't have any "tooltip" component in KTH Style yet 😢. I don't know what is better:

- Not use any tooltip and wait until we have something in KTH Style (can take a lot of time)
- To use this unaccessible tooltip and fix it later
- Make an accessible tooltip component which can be moved to KTH Style in the future

**Review by exacs — APPROVED** [review]

**Comment by exacs** [comment]
The tooltip is good enough. LGTM!

**Pull request merged**